code/drasil-metadata/lib/Data/Drasil/DrasilConfig.hs

```diff
@@ -10,10 +10,22 @@ import Utils.Drasil(drasilConfigPath)
 
 newtype DrasilConfig = DrasilConfig {version :: String} deriving Generic
 
+-- The number of folders that needs to be exited in order to obtain the
```

Comment:
```suggestion
-- The number of folders that need to be exited in order to obtain the
```

Since it is the _folders_ that need to be exited, not the _number_ of folders, the verb should be plural.

Reply:
While we are looking at this comment, "in order to" should just be "to".  :smile:

Reply:
See [this list of low-information content phrases](https://www.webpages.uidaho.edu/range357/extra-refs/empty-words.htm)

Reply:
This is a very good resource! Thank you for that!